client/public/src/components/app.jsx

```diff
@@ -24,31 +24,31 @@ const ProductOptions = () => {
 
   const [buyQty, setBuyQty] = useState(1);
   const [cart, setCart] = useState(0);
-
+  // 18.193.58.249
   // get the products general details such as price, name and reviews
   const getProduct = async (productId) => {
-    const response = await axios.get(`http://18.193.58.249:3002/products/${productId}`);
+    const response = await axios.get(`http://${location.hostname}:3002/products/${productId}`);
     const productData = response.data;
     setProduct(productData);
   }
 
   // get a store's id and location
   const getStore = async (storeId) => {
-    const response = await axios.get(`http://18.193.58.249:3002/stores/${storeId}`);
+    const response = await axios.get(`http://${location.hostname}:3002/stores/${storeId}`);
     const storeData = response.data;
     setStore(storeData);
   }
 
   // get a list of all the stores (names and ids)
   const getStores = async () => {
-    const response = await axios.get(`http://18.193.58.249:3002/stores`);
+    const response = await axios.get(`http://${location.hostname}:3002/stores`);
     const storesData = response.data;
     setStores(storesData);
   }
 
   // get all available stock for the selected item in the selected store
   const getStock = async (productId) => {
-    const response = await axios.get(`http://18.193.58.249:3002/stock/${productId}`);
+    const response = await axios.get(`http://${location.hostname}:3002/stock/${productId}`);
```

Comment:
Is it necessary to have more than just the endpoint here? I would think it wouldn't be required as long as the client and the server are in the same container.